pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -38,8 +37,13 @@ type NodeAffinity struct {
 var _ framework.FilterPlugin = &NodeAffinity{}
 var _ framework.ScorePlugin = &NodeAffinity{}
 
-// Name is the name of the plugin used in the plugin registry and configurations.
-const Name = "NodeAffinity"
+const (
+	// Name is the name of the plugin used in the plugin registry and configurations.
+	Name = "NodeAffinity"
+
+	// ErrReason for node affinity/selector not matching.
+	ErrReason = "node(s) didn't match node selector"
```

Comment:
Should we rename to `ErrReasonSelectorNotMatch`

Reply:
package name is enough to identify the context, externally this looks like: ```nodeaffinithy.ErrReason```